### PR TITLE
Improve setup (#10)

### PR DESCRIPTION
**Pull request by vrumger**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ JosXa/BotListBot is licensed under the MIT License.
 #### Mandatory
 
 - Python 3.7.x
-- A PostgreSQL database instance
+- A PostgreSQL database instance or Docker ([see below](#development-setup))
```
Comment (JosXa) [inline README.md:20]: Nice!

File: README.md [hunk 2]
```diff
@@ -49,6 +49,16 @@ arguments list, and run it again. This will fill the database with some initial,
 1. Run the project via `pipenv run python botlistbot/main.py`
 </details>  
 
+<details>
+<summary>Setup a PostgreSQL database instance with Docker</summary>
+
+1. Install Docker
+1. Run `docker-compose up -d`
+1. Set the `DATABASE_URL` variable in the `.env` file to `postgres://botlistbot:botlistbot@localhost:5432/botlistbot`
```
Comment (JosXa) [inline README.md]: If this goes in template.env I think we're good

Comment (JosXa) [reply README.md]: - [ ] Add to template
- [ ] Remove this step

File: botlistbot/components/basic.py [hunk 3]
```diff
@@ -246,7 +246,7 @@ def register(dp):
     dp.add_error_handler(error)
     dp.add_handler(
         MessageHandler(
-            Filters.text & Filters.group,
+            Filters.text,
```
Comment (JosXa) [inline botlistbot/components/basic.py:249]: Hmm, but why 🤔

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+            - 5432:5432
+        environment:
+            - POSTGRES_USER=botlistbot
+            - POSTGRES_DB=botlistbot
+            - POSTGRES_PASSWORD=botlistbot
+        volumes:
+            - db:/var/lib/postgresql/data
+
+volumes:
+    db:
```
Comment (JosXa) [inline docker-compose.yml:17]: If there's no volume, doesn't that mean all data is lost every time you docker up?

**Review by JosXa — CHANGES_REQUESTED** [review]

File: botlistbot/components/basic.py (hunk 3 above)
Comment (vrumger) [reply botlistbot/components/basic.py:249]: The handler has a check for channel posts which calls another function. That other function adds the channel to the database if the channel isn't already in there.

File: docker-compose.yml (hunk 4 above)
Comment (vrumger) [reply docker-compose.yml:17]: To create a volume, you just need the name and then you specify it in the service.

File: README.md (hunk 2 above)
Comment (vrumger) [reply README.md]: Done! :+1:

File: botlistbot/components/basic.py (hunk 3 above)
Comment (JosXa) [reply botlistbot/components/basic.py:249]: Then why would you filter for group and not (re)move the channel post check?

Comment (vrumger) [reply botlistbot/components/basic.py:249]: The channel post check adds the channel to the database, which is helpful for when you run it for the first time.

Comment (JosXa) [reply botlistbot/components/basic.py:249]: Oh, you removed it. Gotta say, didn't know I had red/green weakness 😅

**Review by JosXa — APPROVED** [review]
LGTM

**Comment by JosXa** [comment]
Conclucdes #10

**Comment by JosXa** [comment]
Thanks a ton :)

**Comment by vrumger** [comment]
Thanks! :+1:

**Pull request merged**